pkg/apis/storage/util/util.go

```diff
@@ -32,4 +32,17 @@ func DropDisabledFields(class, oldClass *storage.StorageClass) {
 			oldClass.AllowedTopologies = nil
 		}
 	}
+	if !utilfeature.DefaultFeatureGate.Enabled(features.ExpandPersistentVolumes) && !allowVolumeExpansionInUse(oldClass) {
+		class.AllowVolumeExpansion = nil
```

Comment:
need to remove the existing code that sets to nil in PrepareForCreate/PrepareForUpdate, right?